public/data/tables/NSO_Bale_Hay_Price_Monthly_Avg2.csv

```diff
@@ -1,19 +1,19 @@
-﻿Month,Average,High,Low
-Month,Average,High,Low
-Dec-18,6750,12000,3200
-Jan-19,6955,10600,3600
-Feb-19,6881,10300,3600
-Mar-19,6505,10000,2700
-Apr-19,6319,10200,3000
-May-19,6456,10400,2250
-Jun-19,6450,10250,3000
-Jul-19,6425,9750,3000
-Aug-19,6967,9750,5500
-Sep-19,5794,10500,3500
-Oct-19,6556,11000,3700
-Nov-19,6876,11000,4000
-Dec-19,7542,11313,4556
-Jan-20,8270,12750,4250
-Feb-20,8252,12000,4000
-Mar-20,8444,13500,4500
-Apr-20,8471,14250,4500
\ No newline at end of file
+Month,data1,data2,data3
+Month,Average,High,Low
```

Comment:
`Month,Average,High,Low` represent column name or a value?

Reply:
Column names. The structure is basically:
First row => keys
Second row => headers